backend/src/main/kotlin/com/infowings/catalog/data/objekt/ObjectService.kt

```diff
@@ -547,6 +564,23 @@ class ObjectService(
         deleteObject(id, username, ::removeOrMark)
     }
 
+    @Suppress("UNCHECKED_CAST")
+    fun recalculateValue(fromMeasureStr: String, toMeasureStr: String, value: BigDecimal): BigDecimal {
+        val fromMeasure = (GlobalMeasureMap[fromMeasureStr] ?: throw RecalculationException("Measure $fromMeasureStr does not exist")) as Measure<DecimalNumber>
+        val toMeasure = (GlobalMeasureMap[toMeasureStr] ?: throw RecalculationException("Measure $toMeasureStr does not exist")) as Measure<DecimalNumber>
+
```

Comment:
It makes sense to keep scope of @Suppress("UNCHECKED_CAST") as small as possible.
At least for this reason consider defining method with following logic:
 
(GlobalMeasureMap[someStr] ?: throw RecalculationException("Measure $fromMeasureStr does not exist")) as Measure<DecimalNumber>

Reply:
For now it is the same as to make all the measure maps generic of the "DecimalNumber", not "*".
So right now I will make all the maps of the generic type `<String, Measure<DecimalNumber>>`